doc/neps/index.rst.tmpl

```diff
@@ -53,12 +53,22 @@ Implemented NEPs
    NEP {{ nep }} — {{ tags['Title'] }} <{{ tags['Filename'] }}>
 {% endfor %}
 
-Defunct NEPs
-------------
+Deferred NEPs
```

Comment:
Is this still a valid status?

Reply:
Otherwise, LGTM.

Reply:
yes, from [nep-0000 workflow](https://github.com/numpy/numpy/blob/master/doc/neps/nep-0000.rst#review-and-resolution) and the [indexing template](https://github.com/numpy/numpy/blob/master/doc/neps/index.rst.tmpl#L56)